spring-rabbit-stream/src/main/java/org/springframework/rabbit/stream/config/SuperStreamBuilder.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2021-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.rabbit.stream.config;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+import org.springframework.util.StringUtils;
+
+/**
+ * Builds a Spring AMQP Super Stream using a fluent API.
+ * Based on <a href="https://www.rabbitmq.com/streams.html">Streams documentation</a>
+ *
+ * @author Sergei Kurenchuk
+ * @since 3.1
+ */
+public class SuperStreamBuilder {
+	private final Map<String, Object> arguments = new HashMap<>();
+	private String name;
+	private int partitions = -1;
+
+	private BiFunction<String, Integer, List<String>> routingKeyStrategy;
+
+	/**
+	 * Creates a builder for Super Stream.
+	 *
+	 * @param name stream name
+	 * @return the builder
+	 */
+	public static SuperStreamBuilder superStream(String name) {
+		SuperStreamBuilder builder = new SuperStreamBuilder();
+		builder.name(name);
+		return builder;
+	}
+
+	/**
+	 * Creates a builder for Super Stream.
+	 *
+	 * @param name       stream name
```

Comment:
```suggestion
	 * @param name stream name
```


Remove extra spaces, we don't align param descriptions by column.